sys/ztimer/Kconfig

```diff
@@ -59,7 +59,7 @@ config MODULE_ZTIMER_MSEC
     bool "Milliseconds"
     select MODULE_ZTIMER
 
-choice
+choice CHOICE_ZTIMER_MSEC_BACKEND
```

Comment:
Otherwise I don't oppose naming this symbol to allow boards to override the default.

Reply:
Hmmm but this also goes against what I originally suggested in the design patterns...  Which one to change?

Reply:
Do we want choices encoded into the symbol?
Do we want the bare minimum `ZTIMER_MSEC_BACKEND`
Do we want to stick to the design pattern with an amendment to use either VARIANT or BACKEND,. whichever is appropriate?

Reply:
Dropping the CHOICE